docs/getting-started/getting-started.md

```diff
@@ -433,15 +433,15 @@ You can see that the UI now shows the movies that are stored in the native appli
 
 ## Adding the Navigation API
 
-We created this API for you for this tutorial--you can view it in the GitHub repository located here: [this GitHub repository](https://github.com/electrode-io/react-native-ernnavigation-api). This very simple API will be used for navigating from the `MovieListMiniApp` to the `MovieDetailsMiniApp`.
+We generated this API for you for this tutorial and the code is hosted at [react-native-ernnavigation-api](https://github.com/electrode-io/react-native-ernnavigation-api) repo. This API will be used for navigating from the `MovieListMiniApp` to the `MovieDetailsMiniApp`.
 
-{% sample lang="android" %}  
-
-1) Mofify the `index.android.js` file as follows so that when selecting a movie in the list, the `MovieListMiniApp` will call the navigation API to navigate to the `MovieDetailsMiniApp` for this movie.
+{% sample lang="android" %}
+1) Modify the `index.android.js` file so that clicking the movie item navigates from  `MovieListMiniApp` to `MovieDetailsMiniApp`.
```

Comment:
mmmm.
Prefer the first version here as well. First we don't really `click` in a mobile app (yes well not my fault if some mobile dev say that or if Android have an onClick method ;P) , we `tap` or we `select` or `press`. `click` is really for web / desktop with my old good mouse :). And not just for that, I think first version works better .